pkgs/by-name/ko/korrect/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  stdenv,
+  fetchCrate,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "korrect";
+  version = "0.1.1";
+
+  src = fetchCrate {
+    inherit (finalAttrs) pname version;
+    hash = "sha256-LS3pzDIb1G6lrP8lAG/Yd7Zcehjm5/TNEvTI+MnOomk=";
+  };
+  cargoHash = "sha256-vpjldD+aVVgCMQ/n+WZDPMMBRFPEeQZa09b45Q3m5UM=";
+
+  # TODO updatescript
+  meta = {
```

Comment:
```suggestion
  nativeBuildInputs = [ installShellFiles ];
  
  postInstall = ''
    installShellCompletion --cmd korrect \
      --bash <($out/bin/korrect completions bash) \
      --fish <($out/bin/korrect completions fish) \
      --zsh <($out/bin/korrect completions zsh)
  '';
  
  meta = {
```

Reply:
Oh, that's how that is done. I'll try that immediately! Thanks!